pages/2.3/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta Server Pages Specification 2.3"
+date: 2019-08-06
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Server Pages defines a template engine for web applications that supports mixing of textual content
+(including HTML and XML) with custom tags, expression language, and embedded Java code, that gets compiled
+into a Jakarta Servlet.
+
+
+* [Jakarta Server Pages 2.3 Specification Document](./pages-spec-2.3.pdf) (PDF)
+* [Jakarta Server Pages 2.3 Specification Document](./pages-spec-2.3.html) (HTML)
+* [Jakarta Server Pages 2.3 Javadoc](./apidocs)
+* [Jakarta Server Pages 2.3 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-pages-tck-2.3.0.zip)
```

Comment:
This needs to be the final download URL, which will be:
https://download.eclipse.org/jakartaee/pages/2.3/eclipse-pages-tck-2.3.0.zip

Reply:
fixed